recipes/ocrmypdf/meta.yaml

```diff
@@ -0,0 +1,73 @@
+{% set name = "ocrmypdf" %}
+{% set version = "11.6.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/ocrmypdf-{{ version }}.tar.gz
+  sha256: 0f624456a50be0b0bc8c0b59704d159f637616c093a1cabe8bb383706561bcf7
+
+build:
+  number: 0
+  skip: true   # [py2k or win]
+  entry_points:
+    - ocrmypdf = ocrmypdf.__main__:run
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - cffi >=1.9.1
```

Comment:
Same here

Reply:
Look at https://github.com/search?p=4&q=org%3Aconda-forge+cffi&type=Code I see that most recipes have cffi in the `host` section. Since I don't know enough to feel comfortable going against the common pattern I prefer to leave this in.

Reply:
If the build passes it will be fine

Reply:
OK, https://github.com/conda-forge/staged-recipes/pull/14066/commits/c1909b54f2f150b9e7a30326cfb26d1e3cee55da passed all the checks, looks like we're good. Thank you!